<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="enterFuncLabel">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Введите задачу Коши, конечное значение расчета и шаг</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="funcLayout" stretch="1,63">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="funcDefineLabel">
        <property name="text">
         <string>F'(x, y) =</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="funcInputField"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="initCondLayout" stretch="1,31,1,31">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="funcInitLabel">
        <property name="text">
         <string>y(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="funcInitInputField">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="funcEqInitLabel">
        <property name="text">
         <string>) = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="funcEqInitInputField"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="calcParamsLayout">
      <item>
       <widget class="QLabel" name="toXLabel">
        <property name="text">
         <string>Построить функцию до x = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="toXInputField"/>
      </item>
      <item>
       <widget class="QLabel" name="stepLabel">
        <property name="text">
         <string>с шагом h = </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stepInputField"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="funcEnterButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="functionGraph"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
